### DISCUSSION TITLE
Is there any way to fetch trophies which use the Trophy type?

### DISCUSSION THREAD

**Discussion by lloydh6** [achievements-app/psn-api#165] [Q&A]
I am trying to understand if there is a way fetch full trophies details via your API implantation? 

I am using the `getUserTrophiesEarnedForTitle` function which works perfectly well but I am missing some of the trophy details I want in the response, namely `trophyName`, `trophyDetail` and `trophyIconUrl`. 

I can see that this is due to the function only returning a `UserThinTrophy[]` for the `trophies` property which has fields picked from the `Trophy` type to create this new `UserThinTrophy`. I am happy to extend this functionality and create a PR but is there there another function that would do this for me? I don't want to make the change if its already implemented elsewhere?

**Comment by wescopeland** [comment]
FWIW, the API is just a thin wrapper around what Sony themselves are returning. If it's not in the response from Sony's server, then psn-api won't be able to return the value.
